### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Gregoirevda**
This is part off this change in serverless-webpack: elastic-coders/serverless-webpack#131
Explain how to work with the serverless-offline, serverless-webpack and serverless-dynamodb-local

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,18 @@ Options are:
 - `--function` or `-f` (required) is the name of the function to run
 - `--path` or `-p` (optional) is a JSON file path used as the function input event
 
+### Using with serverless-offline and serverless-webpack plugin
+
+ Run `serverless offline start`. In comparison with `serverless offline`, the `start` command will fire an `init` and a `end` lifecycle hook which is needed for serverless-offline and serverless-dynamodb-local to switch off both ressources.
```
Comment (hassankhan) [inline README.md]: Typo of 'resources' at the end of the line

Comment (Gregoirevda) [inline README.md]: Thank you. Just deleted the s

**Comment by HyperBrain** [comment]
Agree. I've tested #131 until tomorrow, to check if it does not break using offline without start. Will merge the doc changes after #131 has been merged.

**Comment by hassankhan** [comment]
GTM I think, #131 is merged 👍

**Comment by HyperBrain** [comment]
@hassankhan Yes, you're right.

**Pull request merged**